source/new-starter/how-it-works/index.html.md.erb

```diff
@@ -1,8 +1,56 @@
 ---
-title: How GOV.UK Accounts works
+title: How GOV.UK Account works
 weight: 22
 last_reviewed_on: 2020-12-10
 review_in: 6 months
 ---
 
-# How GOV.UK Accounts works
+# How GOV.UK Account works
+
+The core GOV.UK account architecture includes:
+
+- an authentication service
+- an attribute store
+- a broker
+- a preferences and consent management service
+- a trust framework
+- a fraud protection and monitoring service
```

Comment:
We don't actually have any of these other than the first two implemented yet

Reply:
have amended to be clear about current vs future architecture